docs/source/app-implementing-new.rst

```diff
@@ -0,0 +1,117 @@
+Creating a New App
```

Comment:
`Developing a New App`

Reply:
lol you just agreed to changing this to `Creating..`